apps/dashboard/src/App.vue

```diff
@@ -543,11 +543,15 @@ export default {
 	background-color: var(--color-background-translucent);
 	-webkit-backdrop-filter: var(--background-blur);
 	backdrop-filter: var(--background-blur);
+	opacity: 1 !important;
 
 	&:hover,
 	&:focus,
 	&:active {
-		background-color: var(--color-background-hover);
+		background-color: var(--color-background-hover)!important;
+	}
+	&:focus {
```

Comment:
```suggestion
	&:focus-visible {
```

See https://pawelgrzybek.com/the-difference-between-css-focus-and-focus-visible-pseudo-class/

Reply:
thx @CarlSchwan  :)